src/Sulu/Bundle/LocationBundle/Resources/js/containers/Location/LocationOverlay.js

```diff
@@ -179,7 +179,8 @@ class LocationOverlay extends React.Component<Props> {
         } = this.props;
 
         // enable confirm button if all marker properties are set or no property is set in case of a reset
-        const confirmEnabled = (this.markerLat && this.markerLong) || (!this.markerLat && !this.markerLong);
+        const confirmEnabled = (this.markerLat !== null && this.markerLong !== null)
```

Comment:
This seems not be enough if I reopen the overlay the data is not there. So it seems like there is another check change needed where the data is stored.

Reply:
you are right - should hopefully be fixed now 🙂